script.js

```diff
@@ -94,168 +122,214 @@ const setupRound = (wordPairs, pairRenderLimitIndex) => {
-    if (kanjiClicked === null) {
-      kanjiClicked = clickedElement;
-      kanjiClicked.classList.add("selected");
+  // left or right column
+  const clickedElementsParentElementsClass =
+    clickedElement.parentElement.getAttribute("class");
+  // true if left column was selected, false if right column was selected
+  const elementFromLeftColumnIsSelected =
+    clickedElementsParentElementsClass === "leftColumn";
+  if (elementFromLeftColumnIsSelected) {
+    if (leftColumnElementValueClicked === null) {
+      leftColumnElementValueClicked = clickedElement;
+      leftColumnElementValueClicked.classList.add("selected");
     } else {
-      kanjiClicked.classList.remove("selected");
-      kanjiClicked = clickedElement;
-      kanjiClicked.classList.add("selected");
+      leftColumnElementValueClicked.classList.remove("selected");
+      leftColumnElementValueClicked = clickedElement;
+      leftColumnElementValueClicked.classList.add("selected");
     }
   } else {
-    if (hiraganaClicked === null) {
-      hiraganaClicked = clickedElement;
-      hiraganaClicked.classList.add("selected");
+    if (rightColumnElementValueClicked === null) {
+      rightColumnElementValueClicked = clickedElement;
+      rightColumnElementValueClicked.classList.add("selected");
     } else {
-      hiraganaClicked.classList.remove("selected");
-      hiraganaClicked = clickedElement;
-      hiraganaClicked.classList.add("selected");
+      rightColumnElementValueClicked.classList.remove("selected");
+      rightColumnElementValueClicked = clickedElement;
+      rightColumnElementValueClicked.classList.add("selected");
     }
   }
   // if both values are filled
-  if (kanjiClicked !== null && hiraganaClicked !== null) {
-    const kanji = kanjiClicked.innerHTML;
-    const hiragana = hiraganaClicked.innerHTML;
-    console.log("kanji: " + kanji + " hiragana: " + hiragana)
-    let expectedHiragana = null;
-    // Index of the shuffledKanjiHiraganaGlossaries triple, to take the glossary from
+  if (
+    leftColumnElementValueClicked !== null &&
+    rightColumnElementValueClicked !== null
+  ) {
+    const leftColumnElementValue = leftColumnElementValueClicked.innerHTML;
+    const rightColumnElementValue = rightColumnElementValueClicked.innerHTML;
+    let expectedRightColumnValue = null;
+    // Index of the shuffledLeftValRightValGlossary triple, to take the glossary from
     let glossaryIndex = null;
-    for (let i = lastUsedPairIndex - pairsToRenderCount; i < lastUsedPairIndex; i++) {
-      console.log("i: ", i)
-      console.log("hiragana: " + hiragana + " shuffledKanjiHiraganaGlossaries[i][1]: " + shuffledKanjiHiraganaGlossaries[i][1])
-      if (kanji === shuffledKanjiHiraganaGlossaries[i][0] && hiragana === shuffledKanjiHiraganaGlossaries[i][1]) {
-        expectedHiragana = shuffledKanjiHiraganaGlossaries[i][1];
+    for (
+      let i = lastUsedTripletIndex - pairsToRenderCount;
+      i < lastUsedTripletIndex;
+      i++
+    ) {
```

Comment:
Make sure to format automatically with prettier. You can fine-tune prettier by adding a configuration file to the root of the repository (read more in docs)